auth/server/src/main/java/com/seagates3/controller/IAMController.java

```diff
@@ -368,25 +376,41 @@ ServerResponse serve(FullHttpRequest httpRequest,
           requestAction.equals("GetTempAuthCredentials") ||
           requestAction.equals("UpdateAccountLoginProfile") ||
           requestAction.equals("DeleteAccount"))) {
-      try {
-        if (RootPermissionAuthorizer.getInstance().containsAction(
-                requestAction)) {
-          new IAMApiAuthorizer().authorizeRootUser(requestor, requestBody);
-        } else {
-          new IAMApiAuthorizer().authorize(requestor, requestBody);
-        }
-        LOGGER.info("Request is authorized for user: " + requestor.getName() +
-                    " account: " + requestor.getAccount());
-      }
-      catch (InvalidUserException e) {
-        LOGGER.error(e.getServerResponse().getResponseBody());
-        return e.getServerResponse();
+      // Authorize with IAM Policy
+      serverResponse = new Authorizer().authorize(requestor, requestBody);
+      if (serverResponse != null &&
+          serverResponse.getResponseStatus() != HttpResponseStatus.OK) {
+        return serverResponse;
       }
     }
 
     return performAction(resourceMap, requestBody, requestor);
   }
-
+  /**
+   * Below will load requestor object with user and policy data using ldap calls
+   * @param requestor
+   */
+ private
+  void loadRequestor(Requestor requestor) {
+    try {
+      requestor.setUser(new UserImpl().findByUserId(
+          requestor.getAccount().getName(), requestor.getId()));
+      List<String> policyDocList = new ArrayList<>();
+      if (!Authorizer.isRootUser(requestor.getUser())) {
+        Map<String, Object> dataMap = new HashMap<>();
+        dataMap.put("accountName", requestor.getUser().getAccountName());
```

Comment:
With Rohit changes we have constant file created. We should use constants instead of hardcoded strings

Reply:
Fixed